wifiphisher.py

```diff
@@ -26,7 +26,6 @@
 PORT = 8080
 SSL_PORT = 443
 PEM = 'cert/server.pem'
-PHISING_PAGE = "access-point-pages/minimal"
```

Comment:
There's still a hard-coded value and that's "access-point-pages/". You just need to rename the variable to "PHISHING_PAGES_DIR".